src/contextualHelp/constants.ts

```diff
@@ -12,7 +12,6 @@ export namespace Constants {
 
 export namespace Identifiers {
     export const EmptyFileName = '2DB9B899-6519-4E1B-88B0-FA728A274115';
-    export const GeneratedThemeName = 'ipython-theme'; // This needs to be all lower class and a valid class name.
```

Comment:
Did you test it in other themes? I believe this was used to colorize stuff in things besides dark/light themes.

Reply:
We do'nt need to manually parse these things, else every other extension will have to do this same hacky thing.
Yes, it works with other themes as well.

Reply:
This was only done in jupyter because we were hosting monaco ourselves in a webview and wanted the same styles applied to monaco, at least thats what I recall. Else webviews don't need this to get theming working.

Reply:
Trying to get a screen recording

Reply:
Ok, tried generating it 3 times, the file is over 10mb, since you're testing at your end, please test it.
It works for me.